src/azure-cli/azure/cli/command_modules/hdinsight/_help.py

```diff
@@ -198,6 +209,107 @@
         az hdinsight host restart --resource-group MyResourceGroup --cluster-name MyCluster --host-names hostname1 hostname2
 """
 
+helps['hdinsight autoscale'] = """
+type: group
+short-summary: Manage HDInsight cluster's autoscale configuration.
+"""
+
+helps['hdinsight autoscale create'] = """
+type: command
+short-summary: Enable Autoscale for a running cluster.
+examples:
+  - name: Enable Load-based Autoscale for a running cluster.
+    text: |-
+        az hdinsight autoscale create --resource-group MyResourceGroup --cluster-name MyCluster --type Load \\
+        --min-count 3 --max-count 5
+  - name: Enable Schedule-based Autoscale for a running cluster.
+    text: |-
+        az hdinsight autoscale create --resource-group MyResourceGroup --cluster-name MyCluster --type Schedule \\
+        --timezone "Pacific Standard Time" --days Monday Tuesday --time 09:00 --count 5
+"""
+
+helps['hdinsight autoscale update'] = """
+type: command
+short-summary: Update the Autoscale configuration.
+examples:
+  - name: Update Load-based Autoscale configuration.
+    text: |-
+        az hdinsight autoscale update --resource-group MyResourceGroup --cluster-name MyCluster --max-count 5
+  - name: Update Schedule-based Autoscale configuration.
+    text: |-
+        az hdinsight autoscale update --resource-group MyResourceGroup --cluster-name MyCluster --timezone "China Standard Time"
+"""
+
+helps['hdinsight autoscale show'] = """
+type: command
+short-summary: Get the Autoscale configuration of a specified cluster.
+examples:
+  - name: Get the Autoscale configuration.
+    text: |-
+        az hdinsight autoscale show --resource-group MyResourceGroup --cluster-name MyCluster
+"""
+
+helps['hdinsight autoscale delete'] = """
+type: command
+short-summary: Disable Autoscale for a running cluster.
+examples:
+  - name: Update Load-based Autoscale configuration
```

Comment:
Delete?

Reply:
changed it to "Disable Autoscale for a running cluster."

---
In reply to: [466365224](https://github.com/Azure/azure-cli/pull/14692#discussion_r466365224) [](ancestors = 466365224)